custom/intrahealth/sqldata.py

```diff
@@ -5609,17 +5609,18 @@ def aggregated_data(self):
             'Net à Payer': 0,
         }
         valid_products = self.program_products
+        net_a_payer_set = set()
         for row in rows:
             if valid_products and \
                row['product_id'] not in valid_products:
                 continue
 
             data['Total Facture'] += self.get_value(row['amount_billed'])
-            data['Net à Payer'] += self.get_value(row['amount_owed'])
+            net_a_payer_set.add(self.get_value(row['delivery_amt_owed']))
```

Comment:
Why is this a `set()`?  What if two rows happen to have the same amount owed?  This will result in only unique `delivery_amt_owed` values being summed.